modules/jt-comparing-jenkins-tekton-execution-models.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-comparing-jenkins-tekton-execution-models_{context}"]
+= Comparing Jenkins and Tekton execution models
+
+Jenkins and Tekton offer similar functions, but are architecturally different and execute things differently. This section outlines a brief comparison of the two execution models.
```

Comment:
Optional word change:

The adverbs here might throw off certain users, specifically `architecturally` since it's often used as a noun. Do you think this rephrase makes as much a sense and reads equally well:
`Jenkins and Tekton offer similar functions but are different in architecture and execution.`